docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: model
+title: Bert For Sequence Classification (Biomarker)
+author: John Snow Labs
+name: bert_sequence_classifier_biomarker
+date: 2024-02-13
+tags: [en, licensed, bfsc, biomarker, classification]
+task: Text Classification
+language: en
+edition: Healthcare NLP 5.2.1
+spark_version: 3.0
+supported: true
+engine: tensorflow
+annotator: MedicalBertForSequenceClassification
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is a [BioBERT](https://nlp.johnsnowlabs.com/2022/07/18/biobert_pubmed_base_cased_v1.2_en_3_0.html) based sentence classification model that can determine whether the clinical sentences include terms related to biomarkers or not.
+
+## Predicted Entities
+
+`1`: Contains biomarker related terms
+
+`0`: Doesn't contain biomarker related terms
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/bert_sequence_classifier_biomarker_en_5.2.1_3.0_1707841923776.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/bert_sequence_classifier_biomarker_en_5.2.1_3.0_1707841923776.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+  
+```python
+document_assembler = DocumentAssembler() \
+    .setInputCol('text') \
+    .setOutputCol('document')
+
+sentence_detector = SentenceDetector() \
```

Comment:
Use clinical sentence detector in all clinical model samples.

Reply:
changed with sentence detectordl helathcare model